### DISCUSSION TITLE
[Developer Issue]: fims::Vector operators

### DISCUSSION THREAD

**Discussion by msupernaw** [NOAA-FIMS/FIMS#597] [Developers]
### Issue details

We need to decide at what level fims::Vector operators should be implemented. Do we just allow for implicit conversion to a scalar for vectors of size 1 (original intent), or do we allow full vectorization of vector/vector and vector/scalar operations. Currently, all are implemented and wrapped in def statements.

**Comment by kellijohnson-NOAA** [comment]
This discussion might not be relevant any longer and it did not generate a lot of feedback. I am closing it for now but if questions regarding the development of fims::Vector come up again we can always reopen it.
